packages/api/src/modules/auth/guards/roles.guard.ts

```diff
@@ -3,14 +3,14 @@ import { Reflector } from '@nestjs/core';
 
 import { Role } from '../../../enums/role.enum';
 import { TokenPayload } from '../../../interfaces/token-types';
-import { ROLES_KEY } from '../decorators/roles.decorator';
+import { constants } from '../../../utils';
 
 @Injectable()
 export class RolesGuard implements CanActivate {
   constructor(private reflector: Reflector) {}
 
   canActivate(context: ExecutionContext): boolean {
-    const requiredRoles = this.reflector.getAllAndOverride<Role[]>(ROLES_KEY, [
+    const requiredRoles = this.reflector.getAllAndOverride<Role[]>(constants.roles_key, [
```

Comment:
You could export each constant separately and use index.ts with `export * from './module-with-constant';`
This way you could import  ROLES_KEY directly from '../../../constants' (`import  {ROLES_KEY}  from '../../../constants'`), which would look better, and also uppercase for constants is preffered

Reply:
You could also create a separate file for every const, this is also a nice approach (while some people may argue)